// Error Handling in JS 
// Try-catch 

try{
    let num = 8;
    console.log(num/0);
}
catch(err){
    console.log(err);
}